packages/react-components/src/helpers/format-date.unit.spec.js

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+import {
+  formatDate,
+  isDateAfter,
+  isDateBefore,
+  isDateSameDay,
+} from './format-date';
+
+describe('date functions', () => {
+  it('formats date to DayOfWeek, MonthName DayOfMonth, Year at Hour:Minutes [AM|PM]', () => {
+    const date = formatDate(new Date('December 17, 1995 03:24:00'));
+    expect(date).to.equal('Sunday, December 17, 1995 at 3:24 AM');
+  });
```

Comment:
Date logic is complicated. When I run this locally (on the West coast), it creates a date for 3:24 AM _local time_, then formats it to eastern time, so 3 hours ahead - 6:24 AM.

![image](https://user-images.githubusercontent.com/2008881/146095126-81c4100f-149f-4d9a-9c15-8f65767d9717.png)

In order to avoid this, we may want to create a `new Date()` using a UTC string w/ timezone offset. Another option could be creating a date from a UTC timestamp.